modules/special-rules/pages/a-dying-race.adoc

```diff
@@ -0,0 +1,16 @@
+= A Dying Race
+
+Every Eldar casualty is a tragedy, one step closer to the race's extinction.
```

Comment:
```suggestion
Every Eldar casualty is a tragedy, one step closer to the extinction of that race.
```

Reply:
Why 'that' instead of 'the'? It reads really awkwardly.

Reply:
Mmmm...not sure, brain fart perhaps, or some half-remembered grammar technicality that may or may not be valid...
I'd be happy with "extinction of the race" — though I've had another idea, I'll amend my suggestion.

Reply:
Note that it looks to me as if, when you resolve the conversation, the suggestion won't be available to apply - the buttons get greyed out.